### ISSUE TITLE
Build abruptly ends for publishOnly builds

### ISSUE THREAD

**Issue by tmeasday** [chromaui/chromatic-cli#141]
```
Chromatic CLI v5.0.0-alpha.2
https://github.com/chromaui/chromatic-cli

  ✔ Authenticated with Chromatic
    → Using project token '*******p1sk'
  ✔ Retrieved git information
    → Commit 'b22c8c1' on branch 'master'; found 1 baseline commit
  ✔ Collected Storybook metadata
    → Storybook v5.3.19 for React; supported addons found: Actions, Links
  ✔ Storybook built in 10 seconds
    → View build log at /Users/tom/Chroma/TestApps/test-app-june-1/build-storybook.log
  ✔ Publish complete in 13 seconds
    → View your Storybook at https://5ed474c462cda9002249fc1e-vxhlpwdqav.chromatic.com
  ✔ Published your Storybook
    → View build details at https://www.chromatic.com/build?appId=5ed474c462cda9002249fc1e&number=3
  ↓ Take snapshots of your stories [skipped]
```

It looks like this is perhaps the intended behaviour (maybe it would be good to have a story for the complete output for common cases?), but I feel like the link to the SB + build are a bit buried, and it feels a bit truncated.

# To QA

- Check out the workflow stories
- Run a build that passes (no changes) - should have a nice "Build passed" message
- Disable UI review and UI tests, then run another build - should have a "Storybook published" message

**Comment by tmeasday** [comment]
(simiarly I feel like the links are a bit buried in cases where there are changes or errors too, WDYT @domyen?)

**Comment by domyen** [comment]
I agree with you @tmeasday, those are good points. 

Let's add a story to document publish-only builds, perhaps in the workflows section? wdyt @ghengeveld 

**Messages strawman**
Add a message after the "steps UI" that tells users what to do next depending on what type of build they have. I'm aware that this may duplicate the links posted in each step, but it's also clearer to have a single CTA at the conclusion.  

For example:
- **Publish only:** "Your Storybook was published at https://..."
- **UI Tests, no changes:** "No changes were found in this build. View build details https://..."
- **UI Tests, has changes:** "3 UI changes found. Verify them now https://..." ([this story](https://5d67dc0374b2e300209c41e7-axronkchqy.chromatic.com/?path=/story/cli-messages-errors--build-has-changes) I think)
- **Errors:** "View build details https://..." ([this story](https://5d67dc0374b2e300209c41e7-axronkchqy.chromatic.com/?path=/story/cli-messages-errors--build-has-errors))

**Comment by ghengeveld** [comment]
Done, released in chromatic 5.0.0-beta.7 👉 `npx chromatic@beta`

**Comment by tmeasday** [comment]
Deployed in Release 45
